### PR TITLE
Nav bar

### PR DESCRIPTION
**Pull request by schneiderman805**
### PT: 
https://www.pivotaltracker.com/n/projects/2227584

- creates nav bar at the top of root page

- adds logo to nav bar

- adds a place holder for checkout total

- adds place holders for log in, sign up, and checkout

- adds paths for log in, sign up and checkout links but they have no view yet for this user story

- creates feature file to hold user story and test for nav bar

screenshot 2018-11-29 at 11 49 35

### CODE REVIEW

**Comment by tochman** [comment]
This is a good example of a PR where a screenshot would come in handy

**Review by faraznaeem — CHANGES_REQUESTED** [review]
Please delete unnecessary views and refactor them to haml

File: app/views/layouts/application.html.haml [hunk 1]
```diff
@@ -6,9 +6,11 @@
     = csrf_meta_tags
     = stylesheet_link_tag    'application', media: 'all', data: {'turbolinks-track' => 'reload'}
     = javascript_include_tag 'application', data: {'turbolinks-track' => 'reload'}
+    %link{:href => "https://cdn.jsdelivr.net/npm/tailwindcss/dist/tailwind.min.css", :rel => "stylesheet"}/
```
Comment (faraznaeem) [inline app/views/layouts/application.html.haml]: Hash rockets

File: config/routes.rb [hunk 2]
```diff
@@ -1,9 +1,9 @@
 Rails.application.routes.draw do
-
+  
   devise_for :users
 
   resources :products, only: [:index]
```
Comment (faraznaeem) [inline config/routes.rb]: format code and delete unnecessary lines

**Review by faraznaeem — CHANGES_REQUESTED** [review]

**Review by faraznaeem — APPROVED** [review]

**Pull request merged**